fastapi/encoders.py

```diff
@@ -96,7 +96,7 @@ def jsonable_encoder(
                     or (not key.startswith("_sa"))
                 )
                 and (value is not None or not exclude_none)
-                and ((include and key in include) or key not in exclude)
+                and ((include and key in include) or not exclude or key not in exclude)
```

Comment:
I think this line needs change. When "exclude" is None, "not exclude" is true and "key not in exclude" does not check. 
```suggestion
                and ((include and key in include) or (exclude and key not in exclude))
```

Reply:
you can check my previous commit: https://github.com/tiangolo/fastapi/pull/1730/commits/3be96857187d1e72f21777bf709455c503ead87d I did exactly the same, but it won't work:
There we check do we need this value or not, you suggestion: `If include exists and key in include we need it, or if exclude exists and key not in exists `
But what about case when exclude not exists or empty? In this case we return False and doesn't return value
My logic:
`If include exists and key in include we need it, or if exclude not exists or key not in exists` - It case when exclude not exists or empty.

Reply:
Your logic was excellent.
Now I realized why my change doesn't work.
Thank you for your patch and sorry for my mistakes.

Reply:
No problem :) As I said, I made this mistake too